cgi-bin/Apache/LiveJournal.pm

```diff
@@ -599,68 +637,91 @@ sub trans {
             # the entry is specified but invalid OR
             # the remote user owns the journal we're viewing OR
             # the remote user posted the entry we're viewing
-            my $should_show_page = ( $u && ! $u->is_visible ) ||
-                                   ( $entry && ! $entry->valid ) ||
-                                   ( $remote &&
-                                       ( $remote->can_manage( $u ) || ( $entry && $remote->equals( $poster ) ) )
-                                   );
+            my $should_show_page =
+                   ( $u && !$u->is_visible )
+                || ( $entry && !$entry->valid )
+                || ( $remote
+                && ( $remote->can_manage($u) || ( $entry && $remote->equals($poster) ) ) );
```

Comment:
Possibly misleading indentation on the last line; looks similar to the previous line, but nesting level has increased and indentation has not